src/LabJackPython.py

```diff
@@ -137,7 +139,6 @@ def _loadLibrary():
             wlib = ctypes.CDLL("labjackud")
         if wlib is not None:
             try:
-                wlib.eGetPtr
```

Comment:
This change is not harmless. This code is determining if the eGetPtr function exists and setting _use_ptr appropriately for later use.